# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/seatd/systemd-protection.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- seatd-0.9.2/contrib/systemd/seatd.service.vanilla	2026-01-10 14:39:08.830973558 +0100
+++ seatd-0.9.2/contrib/systemd/seatd.service	2026-01-10 14:39:12.061118593 +0100
@@ -11,7 +11,7 @@
 
 # Filesystem lockdown
 ProtectHome=true
-ProtectSystem=strict
+#ProtectSystem=strict
 ProtectKernelTunables=true
 ProtectControlGroups=true
 PrivateTmp=true
